etc/mqtt_source.yaml

```diff
@@ -1,7 +1,8 @@
 #Global MQTT configurations
 default:
   qos: 1
-  servers: [tcp://127.0.0.1:1883]
+  connectionSelector: mqtt_conf1
```

Comment:
Need to discuss if this could be changed. This my conflict with the docker doc about setting mqtt server with env var